app/_locales/en_GB/messages.json

```diff
@@ -1033,7 +1036,10 @@
     "message": "Type"
   },
   "confirmAccountTypeSmartContract": {
-    "message": "Smart contract account"
+    "message": "Smart account"
```

Comment:
Ideally this changed copy should use a new key so that the translation software can pick up the change

Reply:
This obviously isn't ideal but I'll try flag manually for now for the sake of priority.